src/tools/pontoon/translation_workspace.md

```diff
@@ -192,6 +192,21 @@ Icons to the right indicate the [status](#translation-status) of each translatio
 
 In the screenshot above, the first item is the approved translation (green checkmark), while the other two are rejected suggestions.
 
+#### User banners
+
+User banners provide immediate context about a user to other community members. They indicate either the role of the user, or that the user is new to Pontoon. Banners are displayed at the bottom of the user's avatar.
```

Comment:
Was the reference to administrators [dropped](https://github.com/mozilla-l10n/localizer-documentation/pull/255/commits/2fb90db9df4d4db7457d1a0d456aac5cf271fa18) on purpose here? That's not covered by the other explanation (that's not a role within a project/local).

Reply:
Yes, because "within the current project and locale" was also dropped. 

The text now reads "They indicate either the role of the user..." and "Administrator" is one of the roles:
https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/users.html#user-roles

We could also drop this sentence entirely.

Reply:
Thanks, I think it makes to keep it as is.